sensor/tests/replay/replay_test.go

```diff
@@ -57,7 +57,7 @@ func (suite *ReplayEventsSuite) SetupSuite() {
 		panic(err)
 	}
 	suite.fakeCentral = centralDebug.MakeFakeCentralWithInitialMessages(
-		message.SensorHello("1234"),
+		message.SensorHello("12345678-1234-1234-1234-123456789abc"),
```

Comment:
This is not valid UUIDv4. Here is the validation rule that I found after quick googling: https://stackoverflow.com/a/19989922

Maybe let's use something like this?

```suggestion
		message.SensorHello("00000000-0000-4000-A000-000000000000"),
```

Reply:
... and apply to all 3 occurrences

Reply:
Fair. I'll change it. It's interesting though,`pkg/uuid.go` is not complaining with `12345678-1234-1234-1234-123456789abc` (haven't look at the code)